dotnet/src/SemanticKernel.Core/Functions/KernelFunctionFromMethodOptions.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.ComponentModel;
+using System.Reflection;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.SemanticKernel;
+
+/// <summary>
+/// Optional options that can be provided when creating a <see cref="KernelFunction"/> from a method.
+/// </summary>
+public sealed class KernelFunctionFromMethodOptions
```

Comment:
Since it's `SemanticKernel.Core`, I'm wondering if we should mark this class as `Experimental` for the first period of time. Same for public places where this class is used. This will give us more space if we want to update it later, rename it or fix some issues. In this case we should remember to remove `Experimental` flag after some time.

Reply:
Added, thanks.